docs/ml-guide.md

```diff
@@ -801,6 +801,173 @@ jsc.stop();
 
 </div>
 
+## Example: Model Selection via Train Validation Split
+In addition to  `CrossValidator` Spark also offers
+[`TrainValidationSplit`](api/scala/index.html#org.apache.spark.ml.tuning.TrainValidationSplit) for hyper-parameter tuning.
+It randomly splits the input dataset into train and validation sets based on ratio passed as parameter
+and use evaluation metric on the validation set to select the best model.
+The use is similar to `CrossValidator`, but simpler and less computationally expensive.
+
+`TrainValidationSplit` takes an `Estimator`, a set of `ParamMap`s, and an
+[`Evaluator`](api/scala/index.html#org.apache.spark.ml.Evaluator).
+It begins by splitting the dataset into two parts using *trainRatio* parameter
+which are used as separate training and test datasets. For example with `$trainRatio=0.75$` (default),
+`TrainValidationSplit` will generate training and test dataset pair where 75% of the data is used for training and 25% for validation.
+Similarly to `CrossValidator`, `TrainValidationSplit` also iterates through the set of `ParamMap`s.
```

Comment:
Also, "set of \`ParamMap\`s" -> "set of \`ParamMap\`s provided in the \`estimatorParamMaps\` parameter"